chrome/content/zotero/zoteroPane.js

```diff
@@ -515,6 +515,18 @@ var ZoteroPane = new function()
 					this.highlightTimer = null;
 				}
 				ZoteroPane_Local.collectionsView.setHighlightedRows();
+				return;
+			} else if (event.keyCode == event.DOM_VK_BACK_QUOTE) {
```

Comment:
This was a quick hack I put in and I think I got feedback the backtick might not be the best choice for the toggle (although I quite like it TBH). I believe this needs to be configurable.